src/otx/algorithms/classification/adapters/mmcls/configurer.py

```diff
@@ -648,6 +650,7 @@ def configure_task_adapt_hook(self, cfg, training):
             sampler_flag=sampler_flag,
             sampler_type=sampler_type,
             efficient_mode=efficient_mode,
+            use_adaptive_repeat=True if is_multiclass else False,
```

Comment:
```suggestion
            use_adaptive_repeat=is_multiclass
```
It's a minor one, but how about changing like this?